mne/viz/utils.py

```diff
@@ -535,6 +535,7 @@ def compare_fiff(fname_1, fname_2, fname_out=None, show=True, indent='    ',
     fname_out : str
         The filename used for storing the diff. Could be useful for
         when a temporary file is used.
+
```

Comment:
Why all of these?

Reply:
to comply with pydocstyle [pep257](https://www.python.org/dev/peps/pep-0257/) ... it's cosmetic and should be avoided, I know ... I tried to limit it

Reply:
I don't see it in that spec. One of their example even does not have it:
```
def complex(real=0.0, imag=0.0):
    """Form a complex number.

    Keyword arguments:
    real -- the real part (default 0.0)
    imag -- the imaginary part (default 0.0)
    """
    if imag == 0.0 and real == 0.0:
        return complex_zero
    ...
```
Does `make docstyle` complain for you? It's what we use for checking docstyle

Reply:
yes, it does not seem to be in the spec. I used [pydocstyle](https://github.com/PyCQA/pydocstyle). Apparently pep257 doesn't say anything about this ...

Reply:
Yes we use `pydocstyle` in our CIs but we have specific includes/excludes that we follow. Doing `make docstyle` in the `mne-python` root does the tests that we follow.

Reply:
wouldn't it be better to accept only "one style", either with a blank line before the end of the docstring or without it? Right now it's very mixed across the codebase.

Reply:
Yes at some point. Right now I think we have predominantly followed having no newline.